p2p-tests/maintainer-update-propagation-1-test.ts

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env -S node --require ts-node/register/transpile-only --require tsconfig-paths/register
+
+// Copyright © 2021 The Radicle Upstream Contributors
+//
+// This file is part of radicle-upstream, distributed under the GPLv3
+// with Radicle Linking Exception. For full terms see the included
+// LICENSE file.
+
+import * as path from "path";
+import { strict as strictAssert } from "assert";
+
+import * as source from "proxy-client/source";
+import { sleep } from "ui/src/sleep";
+import {
+  RadicleProxy,
+  UpstreamSeed,
+  commit,
+  getLatestCommitSha,
+  pushRad,
+  radCli,
+  runTestcase,
+  withRetry,
+} from "./lib/p2p";
+
+// Test that updates to a project from the maintainer are replicated to
+// trackers via a seed.
+//
+// In this testcase we stop and start the peers between issuing new commands.
```

Comment:
This is slightly ambiguous since the seed is also a peer and doesn’t convey the intention. I’d suggest the following
```suggestion
// The maintainer and the tracker shutdown and restarted so that they are never connected to the seed at the same time.
```